sympy/combinatorics/pc_groups.py

```diff
@@ -0,0 +1,67 @@
+from sympy.combinatorics.fp_groups import FpGroup, FpSubgroup, subgroup_quotient
+from sympy import sieve
+
+class PcGroup(FpGroup):
+    '''
+    Class implementing Polycyclic groups.
+    To-do : Docstring completion.
+            Add error and exception handling
+    '''
+    def __init__(fp_group, pc_series=None, pcgs=None):
+        self.fp_group = fp_group
+        self.pc_series = pc_series
+        self.pcgs = pcgs
+        self.rel_orders = self.relative_orders()
+
+    def compute_pcgs(self):
+        pcgs = []
+        for i in range(0, len(self.pc_series) - 1):
+            quotient = subgroup_quotient(self.fp_grp.free_group, self.fp_grp,self.pc_series[i], self.pc_series[i+1])
+            pcgs.append(quotient.random())
```

Comment:
Careful: a random element of a cyclic group is not necessarily its generator. For a group of prime order, everything except the identity is in fact a generator but even then you'd need to check that it's not the identity.

In addition to this, will `quotient.random()` return an element of the main group that corresponds to the right generator?

Reply:
> In addition to this, will quotient.random() return an element of the main group that corresponds to the right generator?

The `subgroup_quotient` method returns an `FpGroup` defined on the same generators as that of the parent group `G`(first group in `c_series`).

Reply:
> In addition to this, will quotient.random() return an element of the main group that corresponds to the right generator? 

In addition to this, will quotient.random() return an element of the main group that corresponds to the right generator. So I guess it has to.

Reply:
> In addition to this, will quotient.random() return an element of the main group that corresponds to the right generator?

The `subgroup_quotient` method returns an `FpGroup` which is defined on the generators(of the subset of the generators) of the main group. So, I guess it should.

Reply:
Simply being defined on the same generators isn't enough. For example,
```
F, a, b = free_group("a b")
G = FpGroup(F, [a**2, b**3])
H = FpGroup(F, [b**2, a**3])
```
`G` and `H` are defined on the same generators and they are even isomorphic but `a` had order 2 in `G` and order 3 in `H` so they represent different elements.